env.example

```diff
@@ -0,0 +1,3 @@
+DEBUG=true
+DATABASE_URL=django.db.backends.postgresql://postgres@localhost:5432/postgres
```

Comment:
Is this not supposed to be `DATABASE_URL=django.db.backends.postgresql_psycopg2://postgres@localhost:5432/postgres`. If  you use  `django.db.backends.postgresql` you may need to change the username. Altetnatively you can just use the default values by setting it to `DATABASE_URL=django.db.backends.postgresql`. That should work

Reply:
@Duncanian also see the reason why the travis build id failing. You may need to include the needed environment variables in the travis build environment by adding them through a command in the `.travis.yml file`